lib/model/QubitInformationObject.php

```diff
@@ -1168,6 +1168,21 @@ public function getDigitalObject()
   }
 
   /**
+   * Get the digital object's checksum value
+   *
+   * @return string  digital object checksum or null
+   */
+  public function getDigitalObjectChecksum()
+  {
+    if (null !== $do = $this->getDigitalObject())
+    {
+      return $do->getChecksum();
+    }
+
+    return null;
```

Comment:
Sorry, missed this even though Mike commented on it.  We don't explicitly call `return null` in functions as it's the default behaviour in PHP